includes/classes/admin/admin.php

```diff
@@ -444,7 +444,7 @@ public function should_migrate() {
 	 */
 	public function prev_option_key() {
 		$prefixes = array(
-			'gathercontent-import', // from wordpress.org/plugins/gathercontent-import
+			'content-workflow', // from wordpress.org/plugins/gathercontent-import
```

Comment:
```suggestion
			'content-workflow', // from wordpress.org/plugins/content-workflow
```

Assuming the URL will change as well